NewForce-Cohort-6/tabloidfullstack-mashed-potatoes
# 5PostDetails

# Description
[Trello Ticket #5](https://trello.com/c/gymSElnT)

Added functionality to view an individual post's details on a new page. Post Details include: 
* Title
* Header image (if exists)
* Content
* Publication date)
* Author's Display Name

# Testing Instructions
1. `git add` and `git commit` whatever you're working on on your branch
2. `git checkout main`
3. `git fetch --a`
4. `git checkout 3ViewAllPosts`
5. If you don't see the most recent version of my files, `git pull origin 3ViewAllPosts`
6. If you haven't created the Tabloid database, please do so.
Use this data to fill out tables and columns: [SQL Tables](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/01_Tabloid_Create_DB.sql)
Use this to seed the database : [SQL SEED DATA](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/02_Tabloid_Seed_Data.sql)
7. Click the execute with debugger solid green button to launch the app
8. If you are already logged in to the site, log out.
9. Log into the website using “[gteanby6@craigslist.orgx](mailto:gteanby6@craigslist.orgx)” (this user has 5 posts) 
10. From the home page, click the “My Posts” option in the navbar, then click on one of the post titles
12. You should be redirected to a page with that post's title at the top of the card, and the Header image, Content, Publication date, and Author's Display Name listed below
13. Try the same thing with a post from the “Posts” page on the navbar


# Checklist:
